reports/status.html

```diff
@@ -435,6 +435,7 @@ <h3>Motivation</h3>
       <h2>Declarative custom elements</h2>
       <dl>
         <dt>GitHub Issue(s):</dt>
+        <dd><a href="https://github.com/w3c/webcomponents-cg/issues/32">WICG/webcomponents#187</a></dd>
```

Comment:
I will double check, but does your issue reference the strawman proposal at all (what is linked right below)?  I was looking but didn't see it.

I think an issue for this is good, but I think that would be really important to call out the strawman from the description of your issue and declare if you are building off of it, or this proposal is an entirely net new just to help set context, IMO.

I would also second the comment from Justin in that issue that it would probably make sense to have the conversation over in https://github.com/WICG/webcomponents repo, since that is where the current proposal resides.

Reply:
Also might be good to sync with @EisenbergEffect , who is also making some updates to this section / spec, per https://github.com/w3c/webcomponents-cg/pull/41.  👍

Reply:
@thescientist13 , linked issue #32 is updated with strawman and other links from @EisenbergEffect PR #41

Reply:
> issue that it would probably make sense to have the conversation over in https://github.com/WICG/webcomponents repo

The purpose of the issue is to serve the WCCG tracing around proposal and its status in own format, it does not fit into WICG purpose which is features discussions. I agree that WICG would benefit from summary and progress on particular feature, in slightly different format though. CC-in on Slack.

Reply:
So I think then we would be good if #41 just includes the link to #32 and then it would be part of our current (2022) report?  Then I think we could close this one down since I don't think we're doing any back porting of older reports, but maybe that is something to corroborate with @Westbrook .

Reply:
Yes, please. We can close this once all relevant info is moved to the new PR.